lib/view_models/internet_connectivity_checker.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
+
+import 'package:at_wavi_app/view_models/base_model.dart';
+import 'package:internet_connection_checker/internet_connection_checker.dart';
+
+class InternetConnectivityChecker extends BaseModel {
+  bool isInternetAvailable = false;
+  late StreamSubscription _listener;
+
+  checkConnectivity() async {
```

Comment:
Where is `checkConnectivity` called?
If it's not getting called, `isInternetAvailable ` will always be false.

Reply:
It should be added desktop Home @Sardar1208  ?